Firmware/Marlin.h

```diff
@@ -391,7 +391,9 @@ extern void calculate_extruder_multipliers();
 
 // Similar to the default Arduino delay function, 
 // but it keeps the background tasks running.
-extern void delay_keep_alive(unsigned int ms);
+void _delay_keep_alive(unsigned int ms, bool do_update_lcd);
```

Comment:
Argument default arguments from c++ can be used here instead of macros.
See https://en.cppreference.com/w/cpp/language/default_arguments

Reply:
Oh yes, good point. I wasn't sure if I should modify the parameter list of the original `delay_keep_alive` method.
This is certainly a more elegant solution.